tests/Integration/Tickets/Ignored/IndexTest.php

```diff
@@ -0,0 +1,99 @@
+<?php
+
+namespace Aviator\Helpdesk\Tests\Integration\Tickets\Ignored;
+
+use Carbon\Carbon;
+use Aviator\Helpdesk\Tests\TestCase;
+
+class IndexTest extends TestCase
+{
+    /** @var string */
+    protected $url = 'helpdesk/tickets/ignored';
+
+    /** @test */
+    public function guests_are_redirected_to_login ()
+    {
+        $response = $this->get($this->url);
+
+        $response->assertStatus(302)
+            ->assertRedirect('login');
+    }
+
+    /** @test */
+    public function only_supers_see_ignored_tickets ()
+    {
+        $this->withoutErrorHandling();
+        $user = $this->make->user;
+        $agent = $this->make->agent;
+        $ignoredUser = $this->make->user;
```

Comment:
I like this variable name, $user vs $ignoredUser is better than say $user1, $user2.